extensions/amp-story/1.0/amp-story-shadow-reset.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:host {
+  all: initial !important;
+  color: initial !important;
```

Comment:
Are these sufficient?  Your screenshot looks like it affects `text-transform` and `letter-spacing`; do we need to explicitly reset those as well, or is `color` the only one that's special?

Reply:
The `all` resets everything, but apparently `color` need to be specified. I didn't look too much in the why (it has no default though?), but I tested almost all properties manually.